themes/default/content/blog/serverless-architecture-design/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Serverless Architecture Design with Pulumi"
+date: 2023-10-30T20:40:47Z
+draft: false
+meta_desc: "Learn about the concept of serverless architecture and how to use Pulumi and Go to create your Azure serverless application"
+meta_image: meta.png
+authors:
+    - adora-nwodo
+tags:
+    - serverless
+
+---
+
+Serverless architectures have gained more popularity in the recent years because they make it easier for software developers to focus on application development without the additional operation overhead. With Pulumi, you can deploy and manage your serverless infrastructure and this article shows you how to do this with Azure.
+
+Before we go into the specifics of serverless architecture design with Pulumi on azure, let's go
+through the basics of serverless - this is important for the rest of the article.
+
+## What is Serverless Architecture
+
+Serverless architecture is a cloud computing model that simplifies the management of infrastructure. This allows software developers to focus on writing code without worrying
+about server provisioning, scaling or maintenance. The idea behind it is functions as a service (FaaS), in which the environment dynamically scales in response to demand and code is executed in response to events.
+
+Key characteristics of serverless architecture include:
```

Comment:
its missing a fundamental piece which is "no operations to manage, so no security or firewalls to set. no operating system patching. none of that